utils/markdown/index.js

```diff
@@ -23,7 +28,7 @@ export function formatWithRenderer(text, renderer) {
         mangle: false,
     };
 
-    return marked(text, markdownOptions);
+    return (marked(text, markdownOptions)).trim();
```

Comment:
Why trim the output of the markdown renderer now? I don't know if it will appear any differently in the UI, but I'd like to confirm that

Reply:
It won't change anything in the UI. I just thought it is unnecessary to have white spaces before or after the result string, but I may be wrong. Do you think it would be required in some cases?

Reply:
No, I think this is fine. It also makes the tests nicer as you've seen. I just wanted to make sure that it wasn't accidentally added or anything like that.